install/kubernetes/experimental-install.yaml

```diff
@@ -164,7 +164,6 @@ data:
     drop
     tcp
     flow
-    port-distribution
```

Comment:
for experimental, it might make sense to keep it

Reply:
@glibsm we can turn this back on, but i'd feel a bit safer if we can figure out the underlying issue first https://cilium.slack.com/archives/CQRL1EPAA/p1603439414032600